cmd/world/cardinal/start.go

```diff
@@ -54,6 +60,21 @@ This will start the following Docker services and its dependencies:
 		}
 		cfg.Timeout = -1
 
+		// Replace cardinal log level using flag value if flag is set
+		logLevel, err := cmd.Flags().GetString(flagLogLevel)
+		if logLevel != "" {
+			zeroLogLevel, err := zerolog.ParseLevel(logLevel)
+			if err != nil {
+				return err
```

Comment:
the error from `ParseLevel` isn't particularly nice to return to the user:
<img width="852" alt="image" src="https://github.com/Argus-Labs/world-cli/assets/48813565/e1ab5711-fa14-4754-a75e-72ebc9871c33">

lets return our own:

```go
return fmt.Errorf("invalid value for flag %s: must be one of <list flags here>", flagLogLevel)
```

Reply:
updated